lib/engine/game/g_18_pa/game.rb

```diff
@@ -145,23 +143,86 @@ class Game < Game::Base
                   },
         ].freeze
 
+        SCRANTON_HEX = 'G12'
+        SCRANTON_MARKER_ICON = 'coal'
```

Comment:
The map changes below have mine, not coal icons.
```suggestion
        SCRANTON_MARKER_ICON = 'mine'
```